src/cpp/src/whisper_pipeline_static.hpp

```diff
@@ -25,6 +25,7 @@ class WhisperPipeline::StaticWhisperPipeline : public WhisperPipeline::WhisperPi
 
 private:
     WhisperInitializedModels m_models;
+    std::shared_ptr<ov::Model> m_decoder_model;
```

Comment:
why do we need to store model? once model is compiled, we need to release ov::Model to free memory consumed by its weights

Reply:
We can't compile this model until `generate()` called.

Reply:
It's a temporal solution as wee need to reshape and recompile decoder model for specific number of input tokens, but we know it only on generation stage.